Description: Redirect flash warnings to our wiki, where we can manage users'
  experience, instead of handing them off to people who don't care about them.
Author: Chad MILLER <chad.miller@canonical.com>


Index: dev.vivid/chrome/browser/resources/plugin_metadata/plugins_linux.json
===================================================================
--- dev.vivid.orig/chrome/browser/resources/plugin_metadata/plugins_linux.json	2015-01-15 08:58:48.771045684 -0500
+++ dev.vivid/chrome/browser/resources/plugin_metadata/plugins_linux.json	2015-01-15 08:58:48.767045747 -0500
@@ -87,8 +87,8 @@
     ],
     "lang": "en-US",
     "name": "Adobe Flash Player",
-    "help_url": "https://support.google.com/chrome/?p=plugin_flash",
-    "url": "http://get.adobe.com/flashplayer/",
+    "help_url": "https://wiki.ubuntu.com/Chromium/Getting-Flash#help",
+    "url": "https://wiki.ubuntu.com/Chromium/Getting-Flash",
     "displayurl": true,
     "group_name_matcher": "*Shockwave Flash*"
   },
